etcdserver/api/membership/errors.go

```diff
@@ -21,10 +21,12 @@ import (
 )
 
 var (
-	ErrIDRemoved     = errors.New("membership: ID removed")
-	ErrIDExists      = errors.New("membership: ID exists")
-	ErrIDNotFound    = errors.New("membership: ID not found")
-	ErrPeerURLexists = errors.New("membership: peerURL exists")
+	ErrIDRemoved         = errors.New("membership: ID removed")
+	ErrIDExists          = errors.New("membership: ID exists")
+	ErrIDNotFound        = errors.New("membership: ID not found")
+	ErrPeerURLexists     = errors.New("membership: peerURL exists")
+	ErrMemberPromoted    = errors.New("membership: aleady a voting member")
```

Comment:
ErrMemberNotLearner = errors.New("membership: can only promote a learner member")

Reply:
ok

Reply:
This is not converted to rpc error yet, make they have the same error message.